ibis/common/tests/test_annotations.py

```diff
@@ -123,31 +123,62 @@ def test(a: int, b: int, c: int = 1):
     with pytest.raises(TypeError):
         sig.validate(2, 3, "4")
 
+    args, kwargs = sig.unbind(sig.validate(2, 3))
+    assert args == (2, 3, 1)
+    assert kwargs == {}
 
-def test_signature_from_callable_unsupported_argument_kinds():
-    def test(a: int, b: int, *args):
-        pass
 
-    with pytest.raises(TypeError, match="unsupported parameter kind VAR_POSITIONAL"):
-        Signature.from_callable(test)
+def test_signature_from_callable_with_varargs():
+    def test(a: int, b: int, *args: int):
+        return a + b + sum(args)
```

Comment:
Can you put `...` to indicate that the body of the function doesn't matter, and that what's being tested is the signature? Same for the below tests.

Reply:
Sure, updating.